src/applications/ivc-champva/shared/components/fileUploads/MissingFileOverview.jsx

```diff
@@ -258,16 +263,13 @@ export default function MissingFileOverview({
     .filter(el => el);
 
   // Update sponsor to identify missing uploads
+  const sponsorConditionalPages = getConditionalPages(pages, data);
   const sponsorMiss = {
     name: data?.[nonListNameKey || 'veteransFullName'],
     missingUploads: checkFlags(
-      pages,
+      sponsorConditionalPages,
       data,
-      verifier.identifyMissingUploads(
-        getConditionalPages(pages, data),
-        data,
-        true,
-      ),
+      verifier.identifyMissingUploads(sponsorConditionalPages, data, true),
     ).missingUploads,
```

Comment:
This was where the bug was that showed ghost files on the `MissingFileOverview` component. Originally we were checking all `pages` in the form to find file uploads. Now we're only concerned with pages that are currently in the form flow (conditionally rendered by `depends` funcs)